trace/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal = 1
```

Comment:
You need a newline at the end of this file (I'd recommend setting up your editor so it adds one by default).

Otherwise LGTM.

@chemelnucfin I'd rather not close out the issue, as I believe there is an issue in codegen. But feel free to merge this once you've resolved the newline issue,